Resources/Audio/Effects/attributions.yml

```
@@ -269,3 +269,9 @@
   license: "CC-BY-SA-3.0"
   copyright: "Created by mattroks101 for Bee Station. cig_snuff converted to mono"
   source: "https://github.com/BeeStation/BeeStation-Hornet/pull/29"
+
+- files: ["firefighting_nozzle.ogg"]
+  license: "CC-BY-SA-3.0"
+  copyright: "Fresound 118950, 186950, 790280 composited and edited together by MajorMoth"
+  source: "https://freesound.org/"
+  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)